service.go

```diff
@@ -56,6 +56,37 @@ type IncidentUrgencyRule struct {
 	OutsideSupportHours *IncidentUrgencyType `json:"outside_support_hours,omitempty"`
 }
 
+// ListServiceRulesResponse represents a list of rules in a service
+type ListServiceRulesResponse struct {
+	Offset uint          `json:"offset,omitempty"`
+	Limit  uint          `json:"limit,omitempty"`
+	More   bool          `json:"more,omitempty"`
+	Total  uint          `json:"total,omitempty"`
+	Rules  []ServiceRule `json:"rules,omitempty"`
+}
+
+// ServiceRule represents a Service rule
+type ServiceRule struct {
+	ID         string              `json:"id,omitempty"`
+	Self       string              `json:"self,omitempty"`
+	Disabled   bool                `json:"disabled,omitempty"`
```

Comment:
I was going through the documentation one last time before approving this, and I'm wondering if this needs to be `*bool`. Right now we have this to `omitempty`, which means if the value is `false` it won't be included in the JSON. If the value is `true` it will be present.

I believe this means we'll be able to disable a service rule, but not enable it again because we cannot pass a `false` value in the JSON. Switching this to a `*bool` should address that.

Does that seem reasonable to you?

Reply:
Ah, the bool omitempty problem, I remember seeing this behavior with protobuffers. Yes, that makes total sense, will change the bool references to pointers, thanks for catching that! 👍